packages/atclient/tests/test_atserver_message.c

```diff
@@ -38,10 +38,10 @@ int main() {
   return ret;
 }
 
-#define expect_uint(TEST_TAG, EXP, ACT)                                                                                \
+#define expect_uint(TEST_TAG, EXP, FMT, ACT)                                                                           \
```

Comment:
We use this macro to check multiple types for equality so I added the format specifier as an input.